dist/datasource.ts

```diff
@@ -205,58 +205,60 @@ export default class IrondbDatasource {
       headers['X-Circonus-Account'] = this.accountId;
     }
     headers['X-Snowth-Advisory-Limit'] = isLimited ? this.resultsLimit : "none";
+    headers['Accept'] = 'application/json';
     if (irondbOptions['std']['names'].length) {
+      var paneltype = irondbOptions['std']['names'][0]['leaf_data']['paneltype'] || "Graph";
+      options = {};
+      options.url = this.url;
+      if ('hosted' === this.irondbType) {
+        options.url = options.url + '/irondb';
+        options.url = options.url + '/fetch';
+      }
+      options.method = 'POST';
+      if ('standalone' === this.irondbType) {
+        options.url = options.url + '/fetch';
+      }
+      var metricLabels = [];
+      var interval = irondbOptions['std']['interval'];
+      var start = irondbOptions['std']['start'];
+      var end = irondbOptions['std']['end'];
+      start = Math.floor(start - (start % interval));
+      end = Math.floor(end - (end % interval));
+      var reduce = paneltype === "Heatmap" ? 'merge' : 'pass';
```

Comment:
Can you draw multiple heatmaps at once?  If you can, 'pass' would be right, if not, then 'merge' is really the only option.